.github/workflows/cypress.yml

```diff
@@ -83,23 +83,6 @@ jobs:
         containers: [1, 2, 3, 4, 5, 6, 7, 8]
         php-versions: [ '8.1' ]
         server-versions: [ 'master' ]
-        run-in-parallel:
```

Comment:
`run-in-parallel` is used further down in the workflow, so I think it's worth keeping it for now. Maybe we can change this to `${{ !!github.head_ref }}` again, to make sure that CI runs on development+release branches don't register in the Cypress cloud - and only the PR CI jobs end up in Cypress cloud.